platform/ios/platform/ios/INSTALL.md

```diff
@@ -25,7 +25,7 @@ Before building, follow these steps to install prerequisites:
 1. Install [Homebrew](http://brew.sh)
 1. Install additional dependencies.
    ```bash
-   brew install cmake ccache pkg-config glfw cmake-docs
+   brew install node cmake ccache pkg-config glfw cmake-docs
```

Comment:
1.  Is node required to build?  I don't see it on the GitHub workflows, but maybe it's used for local development.
2. @louwers — maybe we can have the GitHub Workflows match the README, e.g., we should integrate @hallahan's changes into https://github.com/maplibre/maplibre-gl-native/blob/main/.github/workflows/ios-ci.yml#L70 the dependencies in the iOS CI.

Reply:
I'm not sure if it's required. I think it is for running tests at least.

Reply:
The CI dependencies are configured with a separate repo: https://github.com/maplibre/maplibre-ci-runners/blob/main/macos.yaml

So those installation commands should be removed from the workflow.